cmd/up.go

```diff
@@ -106,22 +109,29 @@ List of components that kubeapps up installs:
 			c.SkipGc = false
 		}
 
-		err = c.Run(objs, wd)
-		if err != nil {
+		// mongodb secret
+		// FIXME (tuna): if the mongodb secret exists then do nothing,
+		// otherwise, add it (with new generated rand pw) to the objs list then do full update.
+		if prevsecret, ok, err := mongoSecretExists(c); err != nil {
```

Comment:
I would not use `ok` (it doesn't mean that if it exists or not) better something like `exists`